mongodb/trigger/mongodb/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/project-flogo/datastore-contrib/mongodb/trigger/mongodb
+
+require (
+	github.com/project-flogo/core v0.9.3
+	github.com/project-flogo/datastore-contrib/mongodb/connection v1.0.0
+	go.mongodb.org/mongo-driver v1.1.2
+)
+
+replace github.com/project-flogo/datastore-contrib/mongodb/connection => /Users/bvenkata/Documents/src/flogooss/localflogo/datastore-contrib/mongodb/connection
```

Comment:
We should remove replace statements in all go.mod files

Reply:
Yes, I forgot to remove it. Will do..